dataactvalidator/filestreaming/schemaLoader.py

```diff
@@ -81,16 +84,19 @@ def loadCrossRules(filename):
                 fileId = validationDb.getFileId(record["file"])
                 validationDb.addMultiFieldRule(fileId,record["rule_type"],record["rule_text_one"],record["rule_text_two"],record["description"],record["rule_label"],record["rule_timing"])
 
+    @classmethod
```

Comment:
I really like this revamp of the field and rule file loads. Do you think it makes sense to take it one step further and add top-level path of the broker backend and the path to the validator conifg in `dataactcore/config.py` so we don't have to worry about constructing that path down here?